### DISCUSSION TITLE
PersistentWindows Windows Application event log entries Source is "Application"

### DISCUSSION THREAD

**Discussion by libove** [kangyu-california/PersistentWindows#345] [General]
When PersistentWindows logs entries to the Windows Application event log, it does so with the Source rather generically set to "Application". I believe that it would set the Source to "PersistentWindows", which would make these (many) log entries easier to find/review/filter-out.
![image](https://github.com/user-attachments/assets/c14a861e-ab94-47d0-bd37-846779f6cac6)

**Comment by kangyu-california** [comment]
Register new event source such as "PersistentWindows" would require admin privilege during installation, I can not enforce that requirement for all users. Hence we settle down on pre-existing "Application" as event source, I don't think filter PW event would be a problem if user follow the last section of PW readme home page.
